velox/exec/RowNumber.cpp

```diff
@@ -488,4 +473,74 @@ void RowNumber::spillInput(
   }
 }
 
+BlockingReason RowNumber::isBlocked(ContinueFuture* /* unused */) {
+  const auto reason =
+      yield_ ? BlockingReason::kYield : BlockingReason::kNotBlocked;
+  yield_ = false;
+  return reason;
+}
+
+void RowNumber::addInputInternal(const RowVectorPtr& input, bool fromSpill) {
+  ensureInputFits(input);
+  if (!fromSpill && inputSpiller_ != nullptr) {
```

Comment:
do we expect to encounter this condition in any case? if not, should this be a VELOX_CHECK instead of a no-op return ?